addons/l10n_in/const.py

```diff
@@ -0,0 +1,4 @@
+DEFAULT_IAP_ENDPOINT = "https://l10n-in-edi.api.odoo.com"
```

Comment:
As I ref other modules we set URLs in const.py so still I suggest not moving the method in res_company

Reply:
I noticed that the `payment_*` modules include a const file. Since URLs are being used across other modules, wouldn't it be more consistent and efficient for the URLs to be defined in the constants file rather than within model?